easybuild/easyconfigs/l/libxc/libxc-3.0.0-GCC-5.4.0-2.26.eb

```diff
@@ -13,6 +13,11 @@ toolchainopts = {'lowopt': True}
 
 sources = [SOURCE_TAR_GZ]
 source_urls = ['http://www.tddft.org/programs/octopus/down.php?file=libxc/']
+patches = ['libxc_3_longdouble.patch']
```

Comment:
@micaeljtoliveira Maybe rename the patch file to `libxc-3.0_no-longdouble.patch`, and use `%(version_major_minor)s` here to avoid hardcoding the version (you can then use the same construct in the libxc 4.0 easyconfigs):

```python
patches = ['libxc-%(version_major_minor)s_no-longdouble.patch']

Reply:
See my question above.